src/subcommands/upgrade.rs

```diff
@@ -64,15 +64,15 @@ pub async fn upgrade(
             match result {
                 Ok((downloadable, backwards_compat)) => {
                     progress_bar.println(format!(
-                        "{} {:45} {}",
+                        "{} {:43} {}",
                         if backwards_compat {
                             backwards_compat_msg.store(true, Ordering::Relaxed);
                             YELLOW_TICK.clone()
                         } else {
                             TICK.clone()
                         },
                         mod_.name,
-                        format!("({})", downloadable.filename).dimmed()
+                        format!("{}", downloadable.filename).dimmed()
```

Comment:
You don't need to use a `format!()` here anymore, you can directly do `downloadable.filename.dimmed()`